app/backend/registries/views.py

```diff
@@ -94,9 +95,10 @@ class AuditUpdateMixin(UpdateModelMixin):
     """
 
     def perform_update(self, serializer):
-        serializer.validated_data['update_user'] = (self.request.user.profile.username or
-                                                    self.request.user.profile.name or
-                                                    self.request.user.get_username())
+        if self.request.user.profile.username is None:
+            raise exceptions.ValidationError(('Username must be set.'))
+
+        serializer.validated_data['update_user'] = (self.request.user.profile.username)
```

Comment:
Does it need to be in brackets?
```suggestion
        serializer.validated_data['update_user'] = self.request.user.profile.username
```